src/main/java/com/snowflake/kafka/connector/internal/streaming/SchematizationUtils.java

```diff
@@ -209,6 +209,9 @@ private static String convertToSnowflakeType(Type kafkaType) {
       case BOOLEAN:
         return "BOOLEAN";
       case STRING:
+        if (semanticType != null && semanticType.equals("io.debezium.data.Json")) {
```

Comment:
what's the issue of using VARCHAR in this case?

Reply:
By using varchar you'd have to parse json each time you wanted to pull out a field. With variant can use json.field. So this will skip extra processing.

Reply:
I see, this makes sense, I guess this is a general issue for json value since they will be all mapped to STRING, I will see what we can do here, thanks!